concepts/05 UI Components/Lookup/00 Getting Started with Lookup/15 Bind Lookup to Data.md

```diff
@@ -0,0 +1,418 @@
+Lookup can load data from different data source types. To use a local array, assign it to the [dataSource](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#dataSource) property. If array elements are objects, set the fields that supply Lookup's value ([valueExpr](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#valueExpr)) and displayed value ([displayExpr](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#displayExpr)). For information on other data source types, refer to the following articles:
```

Comment:
Don't use {WidgetName} if the article does not participate in inheritance.

Reply:
Noted